spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -20,6 +20,9 @@
 		<mockwebserver.version>0.0.12</mockwebserver.version>
 		<restassured.version>3.0.2</restassured.version>
 		<spock-spring.version>1.0-groovy-2.3</spock-spring.version>
+		<version.jaeger>0.19.0</version.jaeger>
+		<io.opentracing.version>0.22.0</io.opentracing.version>
```

Comment:
I will update the version as jaeger-core - 0.19.0 has a dep with opentracing 0.22

```
[INFO] com.uber.jaeger:jaeger-core:jar:0.19.0
[INFO] +- com.uber.jaeger:jaeger-thrift:jar:0.19.0:compile
[INFO] |  \- org.apache.thrift:libthrift:jar:0.9.2:compile
[INFO] |     +- org.apache.httpcomponents:httpclient:jar:4.2.5:compile
[INFO] |     |  +- commons-logging:commons-logging:jar:1.1.1:compile
[INFO] |     |  \- commons-codec:commons-codec:jar:1.6:compile
[INFO] |     \- org.apache.httpcomponents:httpcore:jar:4.2.4:compile
[INFO] +- io.opentracing:opentracing-api:jar:0.22.0:compile
[INFO] +- com.google.code.gson:gson:jar:2.8.0:compile
[INFO] +- org.slf4j:slf4j-api:jar:1.7.16:compile
```